src/views/Dashboard.vue

```diff
@@ -129,7 +136,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
             </v-list-item-avatar>
             <v-list-item-content>
               <v-list-item-title class="title font-weight-light">
-                Cylc UI Quickstart
+                Cylc UI Quickstart2
```

Comment:
?

Reply:
Sorry! Changing the page as my reloads were caching even with ctrl F5.